src/distro/distro.py

```diff
@@ -1062,6 +1062,13 @@ def _parse_os_release_content(lines: TextIO) -> Dict[str, str]:
                 # Ignore any tokens that are not variable assignments
                 pass
 
+        if "version" in props:
+            # extract release codename from the the version
+            match = re.search(r"(\(\D+\))|,(\s+)?\D+", props["version"])
```

Comment:
Since that code was just moved, maybe it's out of scope here?: The second half of the regex pattern — `|,(\s+)?\D+` — seems weird:
- I cannot find any test cases with pattern like that (was looking with `git grep ', *[0-9]\+'`).
- Any match to that second half will violate the "with parentheses" assumption below.
- `(\s+)?` looks like an overcomplicated version of `(\s*)`.

Reply:
> Since that code was just moved, maybe it's out of scope here?

I don't think it is : I only moved it two blocks above to let #230 logic properly overwrite `props["codename"]` as expected.
I didn't want to bring an affectation overhead for this fix, so I've de-duplicated it too.

>The second half of the regex pattern — `|,(\s+)?\D+` — seems weird:
> 
>     * I cannot find any test cases with pattern like that (was looking with `git grep ', *[0-9]\+'`).
> 
>     * Any match to that second half will violate the "within parentheses" assumption below.
> 
>     * `(\s+)?` looks like an overcomplicated version of `(\s*)`.

I've thought too, I am OK to improve it while tweaking those lines :+1:

Reply:
Oh damn, I was reading `\D` for `\d` :man_facepalming: Sorry! Completely different picture now:

```console
# git --no-pager grep 'VERSION.*, *[^0-9]\+' -- tests/resources/distros/
tests/resources/distros/linuxmint17/etc/os-release:VERSION="14.04.3 LTS, Trusty Tahr"
tests/resources/distros/ubuntu14/etc/os-release:VERSION="14.04.3 LTS, Trusty Tahr"
```